src/main/app/case/from-api-format.ts

```diff
@@ -31,6 +31,7 @@ const checkboxConverter = (value: string | undefined) => {
 
 const fields: FromApiConverters = {
   ...invert(formFieldsToCaseMapping),
+  ...invert(readOnlyFormFieldsToCaseMapping),
```

Comment:
My memory is bad but i think there was a reason this was removed (i believe it was here before).

Reply:
well, lets find out